Cython/Compiler/Code.py

```diff
@@ -715,17 +716,17 @@ def put_code(self, output):
         if self.init:
             writer = output['init_globals']
             writer.putln(f"/* {self.name}.init */")
-            if isinstance(self.init, str):
-                writer.put(self.format_code(self.init))
-            else:
-                self.init(writer, output.module_pos)
```

Comment:
I'm a little worried I might have lost something here - it isn't clear to me what would cause `self.init` to be callable

Reply:
Not sure either. Might have been a feature at some point, but the test suite seems to be happy without it.